lib/src/rewrite.rs

```diff
@@ -587,6 +587,11 @@ impl<'settings, 'repo> DescendantRebaser<'settings, 'repo> {
                 new_commit,
             }));
         }
+        Ok(None)
+    }
+
+    pub fn rebase_all(&mut self) -> Result<(), TreeMergeError> {
+        while self.rebase_next()?.is_some() {}
         // TODO: As the TODO above says, we should probably change the API. Even if we
         // don't, we should at least make this code not do any work if you call
         // rebase_next() after we've returned None.
```

Comment:
This part of the sentence can now be removed, right?

Reply:
Good point, thanks!